test/codec-utils.test.js

```diff
@@ -0,0 +1,28 @@
+import Qunit from 'qunit';
+import { translateLegacyCodecs } from '../src/codec-utils';
+
+const { deepEqual, module, test } = Qunit;
```

Comment:
As awesome and clean as I think this is, we don't do this any of the other tests and if we make the jump to QUnit 2.0 in this project, it might make the transition slightly more annoying